### PR TITLE
Fix logic error in sendalerts.py

### PR DESCRIPTION
**Pull request by JaymesKat**
Removing a double initialization on line 22

## Status
**READY**

## Migrations
NO

## Description
Fix an error in sendalerts.py file

## Related PRs
branch | PR
------ | ------
Ft alert check running often 158174593  | [link](https://github.com/andela/hc-aces-kla/pull/18)
Ft nags user until resolved #158174590  | [link](https://github.com/andela/hc-aces-kla/pull/10)

### CODE REVIEW

File: hc/api/management/commands/sendalerts.py [hunk 1]
```diff
@@ -48,7 +48,9 @@ def handle_many(self):
         if repeat_list_approved:
             checks = (
                 list(
-                    running_checks.iterator()) +
+                    going_down.iterator()) +
+                list(
+                    going_up.iterator()) +
                 list(repeat_list_approved))
         else:
```
Comment (archbishop12) [inline hc/api/management/commands/sendalerts.py]: Add the going up and going down in the else statement also.

Comment (JaymesKat) [reply hc/api/management/commands/sendalerts.py]: @archbishop12 I have made the additions to the else block as pointed put

**Comment by JaymesKat** [comment]
@EdwinKato This has been tested locally and it works. Please merge

**Comment by EdwinKato** [comment]
Hi @JaymesKat any chance you could close this PR? It shows up in some of my lists

**Comment by JaymesKat** [comment]
> Hi @JaymesKat any chance you could close this PR? It shows up in some of my lists

Closed.

**Pull request closed without merging**